proxies/go/handler_client.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/devcyclehq/go-server-sdk"
+	"net/http"
+)
+
+type clientRequestBody struct {
+    ClientId         string   `json:"clientId"`
+    SdkKey      string   `json:"sdkKey, omitempty"`
+    EnableCloudBucketing bool `json:"enableCloudBucketing"`
+    WaitForInitialization bool `json:"waitForInitialization"`
+    Options devcycle.DVCOptions `json:"options"`
+}
+
+type clientResponseBody struct {
+    Message         string   `json:"message, omitempty"`
+    AsyncError      string   `json:"asyncError, omitempty"`
+    Exception       string   `json:"exception, omitempty"`
+}
+func clientHandler(w http.ResponseWriter, r *http.Request) {
+    var reqBody clientRequestBody
+    err := json.NewDecoder(r.Body).Decode(&reqBody)
+    if err != nil {
+        http.Error(w, err.Error(), http.StatusBadRequest)
+        return
+    } else if reqBody.ClientId == "" {
+        http.Error(w, "Invalid request: missing clientId", http.StatusBadRequest)
+        return
+    }
+
+    reqBody.Options.DisableLocalBucketing = reqBody.EnableCloudBucketing
+    var lb *devcycle.DevCycleLocalBucketing
+    var res clientResponseBody
+    if reqBody.EnableCloudBucketing == false {
+        lb, err = devcycle.InitializeLocalBucketing(reqBody.SdkKey, &reqBody.Options)
+        if err != nil {
+           res.AsyncError = err.Error()
```

Comment:
this should be `exception` cause I don't think the go proxy like the dotnet one should return an `asyncError` (see slack for details). You can also add a check now for `awaitForInitialization` and then just put a TODO in that if/else branch until we we implement async init on the sdk.

Reply:
ill change the asyncError to be exception, but I dont think we know how the awaitForInitialization is going to work quite yet so I dont think theres any point in a TODO

Reply:
kk sounds good